docs/whatsnew-5.2.rst

```diff
@@ -391,3 +391,9 @@ a TLS connection to ArangoDB result backend.
 
 If you'd like to do so, set the ``verify`` key in the
 :setting:`arangodb_backend_settings` dictionary to ``True``.
+
```

Comment:
this will be added to v5.3.0, so this part should be removed and find a location in CLI part of the doc